src/v2/Apps/Search/Routes/Artworks/index.tsx

```diff
@@ -1,7 +1,6 @@
 import { RouterState, withRouter } from "found"
 import React from "react"
 import { ArtworkFilter_viewer } from "v2/__generated__/ArtworkFilter_viewer.graphql"
-import { ZeroState } from "v2/Apps/Search/Components/ZeroState"
```

Comment:
I believe this custom `ZeroState` component also contains the feedback form. I'm not sure whether we want to keep or remove that - I'll ask in Slack.